INN/umbrella-wcij
# Replace newsletter signup form in header with a widget area that can be used for whatever

## Changes

This pull request makes the following changes:

- Removes the form in the header and replaces it with a new widget area.

This screenshot shows it with the blue and grey button options.
![Screen Shot 2020-04-01 at 16 51 01](https://user-images.githubusercontent.com/1754187/78187928-5a000800-743d-11ea-9d40-f36bf1c7cdcb.png)


## Why



For #15

## Testing/Questions

Features that this PR affects:

- The area to the right of the site logo.


Questions that need to be answered before merging:

- [ ] Should we remove all the styles for what was formerly the form up there?
- [ ] Do they want any other style changes to this area?

Steps to test this PR:

0. Check out this branch.
1. Add a Custom HTML widget to the Header Newsletter Signup widget area, with the title "Subscribe to our newsletters" and widget content of:
```html
<a class="btn" href="/subscribe/">Subscribe! btn</a><br/>
<a class="btn btn-primary" href="/subscribe/">Subscribe! btn-primary</a>
```
